src/libs/ValidationUtils.js

```diff
@@ -321,7 +321,19 @@ function isValidRoutingNumber(number) {
  * @returns {Boolean[]}
  */
 function doesFailCharacterLimit(maxLength, valuesToBeValidated) {
```

Comment:
We can remove this one now that all usages of it are updated to use `doesFailCharacterLimitAfterTrim` right?